tests/mq/plugins/test_broFixup.py

```diff
@@ -0,0 +1,1697 @@
+            "cert_chain_fuids":["Fo4Xkx1WrJPQJVG6Zk","FZcDnY15qCFTlPt0E7"],
+            "client_cert_chain_fuids":[],
+            "subject":"CN=geo.mozilla.org,OU=WebOps,O=Mozilla Foundation,L=Mountain View,ST=California,C=US",
+            "issuer":"CN=DigiCert SHA2 Secure Server CA,O=DigiCert Inc,C=US",
+            "validation_status":"ok",
+            "pfs":'true',
+            'category': 'bro',
+            'type': 'ssl',
+            'customendpoint': ' '
+        }
+
+        result, metadata = self.plugin.onMessage(event, self.metadata)
+        self.verify_defaults(result)
+        self.verify_metadata(metadata)
+        assert toUTC(event['ts']).isoformat() == result['utctimestamp']
+        assert toUTC(event['ts']).isoformat() == result['timestamp']
+        assert sorted(result['details'].keys()) == sorted(event.keys())
+        assert result['summary'] == 'SSL: 36.70.241.31 -> 63.245.215.82:443 geo.mozilla.org'
+    
+    def test_ssl_log2(self):
+        event = {
+            "ts":1502751597.597052,
+            "uid":"CWmwax23B9dBtn3s16",
+            "sourceipaddress":"36.70.241.31",
+            "sourceport":49322,
+            "destinationipaddress":"63.245.215.82",
+            "destinationport":443,
+            "version":"TLSv12",
+            "cipher":"TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
+            "curve":"secp256r1",
+            "resumed":'false',
+            "established":'true',
+            "cert_chain_fuids":["Fo4Xkx1WrJPQJVG6Zk","FZcDnY15qCFTlPt0E7"],
+            "client_cert_chain_fuids":[],
+            "subject":"CN=geo.mozilla.org,OU=WebOps,O=Mozilla Foundation,L=Mountain View,ST=California,C=US",
+            "issuer":"CN=DigiCert SHA2 Secure Server CA,O=DigiCert Inc,C=US",
+            "validation_status":"ok",
+            "pfs":'true',
+            'category': 'bro',
+            'type': 'ssl',
+            'customendpoint': ' '
+        }
+
+        result, metadata = self.plugin.onMessage(event, self.metadata)
+        self.verify_defaults(result)
+        self.verify_metadata(metadata)
+        assert toUTC(event['ts']).isoformat() == result['utctimestamp']
+        assert toUTC(event['ts']).isoformat() == result['timestamp']
+        assert sorted(result['details'].keys()) == sorted(event.keys())
+        assert 'server_name' in result['details']
+        assert result['summary'] == 'SSL: 36.70.241.31 -> 63.245.215.82:443 63.245.215.82'
+
+    def test_dhcp_log(self):
+        event = {
+            "ts":1505701256.181043,
+            "uid":"Cbs59u2x6KXu85dsOi",
+            "sourceipaddress":"10.26.40.65",
+            "sourceport":68,
+            "destinationipaddress":"10.26.40.1",
+            "destinationport":67,
+            "mac":"00:25:90:9b:67:b2",
+            "assigned_ip":"10.26.40.65",
+            "lease_time":86400.0,
+            "trans_id":1504605887,
+            'category': 'bro',
+            'type': 'dhcp',
+            'customendpoint': ' '
```

Comment:
Does fluentd set the customendpoint to an empty string? Does it make more sense to instead of using a ' ', populate it with some value?

Reply:
It can populate with customendpoint = bro